packages/flutter_tools/test/general.shard/base/async_guard_test.dart

```diff
@@ -288,7 +287,7 @@ void main() {
         if (!completer.isCompleted) {
           completer.complete(null);
         }
-      }));
+      })!);
```

Comment:
what are these ones for 🤔

Reply:
`unawaited` takes a nonnull `Future` and `runZonedGuarded` can be null.

However a better change is to update out `unawaited` to `void unawaited(Future<void>? future)` and match the signature in https://pub.dev/documentation/pedantic/latest/pedantic/unawaited.html
https://github.com/flutter/flutter/blob/a2f67720ef146ec44a35800706128179c33e43a9/packages/flutter_tools/lib/src/base/common.dart#L36

Good eye.